internal/provider/gateway_name.go

```diff
@@ -101,3 +108,24 @@ func syncContractsNameGateways(d *schema.ResourceData, gw *workloads.GatewayName
 	d.SetId(fmt.Sprint(gw.ContractID))
 	return
 }
+
+// validateBackends ensures that if tlsPassthrough is set, the backend form is ip:port
+// and if not set, the form is http://ip[:port]
+func validateBackends(backnends []zos.Backend, tlsPassthrough bool) error {
+	for _, u := range backnends {
+		if tlsPassthrough {
+			_, _, err := net.SplitHostPort(string(u))
```

Comment:
`SplitHostPort` will split `http://1.1.1.1` to `http` and `//1.1.1.1` without errors, this is not valid, right?

Reply:
indeed. will fix it